userstorage/__init__.py

```diff
@@ -11,3 +11,8 @@
 # Helpers.
 from userstorage.options import missing_handler
 from userstorage.config import load_config
+
+# Errors.
+from userstorage.errors import Unsupported
+from userstorage.errors import CreateFailed
+from userstorage.errors import Missing
```

Comment:
Looks good, and with this you don't need to document the errors module.

Reply:
True, I'll change the order of the commits.